website/source/docs/providers/aws/r/placement_group.html.markdown

```diff
@@ -32,3 +32,11 @@ The following arguments are supported:
 The following attributes are exported:
 
 * `id` - The name of the placement group.
+
+## Import
+
+Placement groups can be imported using the `name`, e.g. 
+
+```
+terraform import aws_placement_group.prod_pg production-placement-group
```

Comment:
can we prefix these with `$` ? That would be consistent with the rest of our docs:

```console 
$ terraform
usage: terraform [--version] [--help] <command> [<args>]
```

from https://www.terraform.io/docs/commands/index.html